packages/gatsby-image/withIEPolyfill/index.d.ts

```diff
@@ -0,0 +1,13 @@
+import * as React from "react"
+
+import GatsbyImage, { GatsbyImageProps } from "../index"
+
+interface GatsbyImageWithIEPolyfillProps extends GatsbyImageProps {
+  objectFit?: `fill` | `contain` | `cover` | `none` | `scale-down`
+  objectPosition?: string
+}
+
+export default class GatsbyImageWithIEPolyfill extends React.Component<
+  GatsbyImageWithIEPolyfillProps,
+  any
```

Comment:
The [`ImageWithIEPolyfill` component](https://github.com/janKir/gatsby/blob/topics/gatsby-image-ie-polyfill-types/packages/gatsby-image/src/withIEPolyfill/index.js) doesn't seem to have any internal state, so we can remove this second generic.

Reply:
I removed the state generic.